src/main/java/io/buildpacks/example/sample/PeerReview.java

```diff
@@ -1,7 +1,4 @@
-// PeerReview.java
-
-public class PeerReview {
```

Comment:
3. Consider adding more tests to cover edge cases like very large or very small numbers, zeros, or cases where the sum exceeds the range of `int`.